presto-main/src/main/java/com/facebook/presto/dispatcher/QueryPrerequisitesManager.java

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Maybe moved these `prerequisites` class into a different package?

Reply:
Moving these to `presto.prerequisites` package